src/main/java/frc/robot/sensors/ElevatorLight.java

```diff
@@ -23,6 +25,7 @@
 public class ElevatorLight extends SubsystemBase {
 
   private CANdle candle;
+  private Timer timer = new Timer();
```

Comment:
This timer is never started, so it never measures the specified time as having elapsed. The timer (which should probably be moved to `Controls.java`) should be started when we enter teleop less than five seconds after leaving auto and stopped and reset when we leave teleop. (`RobotModeTriggers.autonomous().debounce(5).and(RobotModeTriggers.teleop())` produces a trigger that is true while in teleop less than 5 seconds after being in auto for more than 5 seconds.)